community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/ParallelSort.java

```diff
@@ -166,11 +164,11 @@ public ParallelSort( Radix radix, LongArray dataCache, long highestSetIndex,
         return rangeParams;
     }
 
-    private String dumpBuckets( int[][] rangeParams, int[] bucketRange, int[] bucketIndex )
+    private String dumpBuckets( long[][] rangeParams, int[] bucketRange, long[] bucketIndex )
```

Comment:
Aha, okay, I thought this was on the order of significant fractions of the imported dataset.